python_modules/dagster/dagster/_config/pythonic_config/config.py

```diff
@@ -79,25 +80,25 @@ def _is_frozen_pydantic_error(e: Exception) -> bool:
     )
 
 
-class MakeConfigCacheable(BaseModel):
+class MakeConfigCacheable(DagsterModel):
```

Comment:
whats up here? it seems like we set the config on this object so inheriting doesn't really do anything but i assume im missing something

Reply:
oops thanks for catching, that's a relic of an earlier approach I took. will revert.